features/facebookImport/src/model/importers/utils/importer-util.js

```diff
@@ -26,7 +26,12 @@ async function readFullPathJSONFile(fullEntryName, zipFile) {
         throw new MissingFileImportException(fullEntryName);
     }
 
-    const rawContent = await zipFile.getContent(dataZipEntry);
+    // TODO: We should not need the lines above
```

Comment:
Just to be clear, the only change here is the comment, right? And you mean *below* here?

Reply:
Ideally in this case we should use the code in the comment instead of the one above the comment. Just at the moment because file path differ between android and the browse pod it does not work.

Reply:
Er... the one above will only show in these diffs windows. Context will be lost when merged.

Reply:
Added for now a hack to handle different ids between mobile and the browser pod. Like this all the speed improvements are active.